q2_types/feature_data/tests/test_format.py

```diff
@@ -132,6 +132,13 @@ def test_tsv_taxonomy_directory_format(self):
 
         format = TSVTaxonomyDirectoryFormat(self.temp_dir.name, mode='r')
 
+        filepath = self.get_data_path('aligned-dna-sequences.fasta')
```

Comment:
Oops, this looks like some copy-pasta (?), and appears to be the [reason travis is failing](https://travis-ci.org/qiime2/q2-types/builds/517441639?utm_source=github_status&utm_medium=notification)

Reply:
🤦‍♂️